### DISCUSSION TITLE
Wrong peer dependency check?

### DISCUSSION THREAD

**Discussion by fehnomenal** [pnpm/pnpm#4563] [Q&A]
If I try to install `@tailwindcss/typography` in my project with `tailwindcss` already installed I get the following warning:

    WARN  @tailwindcss/typography@0.5.2 requires a peer of tailwindcss@>=3.0.0 || >= 3.0.0-alpha.1 || insiders but version 3.0.24 was installed.

This does not seem right or am I doing something wrong?

My `package.json` contains:
```json
  "devDependencies": {
    "@tailwindcss/typography": "~0.5.2",
    "tailwindcss": "~3.0.24",
  }
```

**Comment by zkochan** [comment]
`>=3.0.0 || >= 3.0.0-alpha.1 || insiders` is an invalid semver range. Does it work with npm or Yarn?

https://runkit.com/embed/mowt5sl5egle

**Reply by NickDarvey** [reply]
It looks like they've [revised the semver range](https://github.com/tailwindlabs/tailwindcss-typography/commit/66baf9bdf75663732168cced74beb3f98403c864) but haven't released the change yet. @fehnomenal did you find a workaround?

**Reply by NickDarvey** [reply]
Oh, it's the `|| insiders` bit which doesn't work. :(

**Reply by NickDarvey** [reply]
@zkochan, it works fine with NPM 6.14.13.

Tags are [valid in dependencies](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#dependencies) so maybe handles tags outside of semver?

**Reply by zkochan** [reply]
I don't see a warning with the latest pnpm.
